### ISSUE TITLE
[Dispatcher] DiffbrowsersTask and DiffenatorTask: UNKNOWN: Not found family by name "Solway"

### ISSUE THREAD

**Issue by graphicore** [googlefonts/fontbakery-dashboard#119]
Currently, the family "Solway"  is in [google/fonts master git](https://github.com/google/fonts/tree/master/ofl/solway) but not available via the [google fonts production api](https://fonts.google.com/specimen/Solway) (gives "Page not found" **at the moment**)

[In the dispatcher](https://fontbakery.graphicore.de/dispatcher/process/f139fe33-e6ea-418f-a28a-4665884d4580) this leaves to a badly handled error:

i.e. for Task: Diffbrowsers

```
Method: activate failed

Error: 2 UNKNOWN: Not found family by name "Solway"
    at Object.exports.createStatusError (/var/javascript/node_modules/grpc/src/common.js:91:15)
    at Object.onReceiveStatus (/var/javascript/node_modules/grpc/src/client_interceptors.js:1204:28)
    at InterceptingListener._callNext (/var/javascript/node_modules/grpc/src/client_interceptors.js:568:42)
    at InterceptingListener.onReceiveStatus (/var/javascript/node_modules/grpc/src/client_interceptors.js:618:8)
    at callback (/var/javascript/node_modules/grpc/src/client_interceptors.js:845:24)
```

The error should be handled gracefully and give a clear descriprion.

**Comment by graphicore** [comment]
Related #118 #120

**Comment by graphicore** [comment]
Fixed in 62d5a14
